php/php-rr-sdk.md

```diff
@@ -186,41 +169,46 @@ use Temporal\Workflow;
 
 class UploadWorkflow
 {
-    public function run(Workflow\ContextInterface $ctx, string $input): string
+    public function run(string $input): string
     {
-        $promise = yield new Workflow\ExecuteActivy(
+        $promise = new Workflow\ExecuteActivy(
             'activityName',
             $input, 
-            100 // startToFinishTimeout, etc. 
+            [
+                ExecuteActivy::START_TO_FINISH_TIMEOUT   => 100,        
+                ExecuteActivy::SCHEDULE_TO_START_TIMEOUT => 10
```

Comment:
I would stick to the same names the service uses:

SCHEDULE_TO_START
START_TO_CLOSE
SCHEDULE_TO_CLOSE